app/javascript/components/pages/Donate.jsx

```diff
@@ -0,0 +1,104 @@
+import SharedLayout from 'components/layout/SharedLayout';
+import Banner from 'components/Banner';
+import Button from 'components/Button';
+import PageTitleWithContainer from 'components/PageTitleWithContainer';
+import { testDonationAmounts, productionDonationAmounts } from 'datasources';
+
+import 'stylesheets/page';
+import 'stylesheets/donate';
+
+const Donate = () => {
+    const [selectedPrice, setSelectedPrice] = useState(100);
+
+    return (
+        <SharedLayout>
+            <PageTitleWithContainer text="Donate" />
+            <div className="flex flex-row p-12 justify-center flex-wrap">
+                <div className="flex flex-col mb-20 md:mb-0 md:mr-20">
+                    <h2 className="text-2xl font-medium mb-8">Choose a donation amount</h2>
+                    <DonationAmounts
+                        selectedPrice={selectedPrice}
+                        setSelectedPrice={setSelectedPrice}
+                    />
+                </div>
+                <div className="max-w-[25rem]">
+                    <h2 className="text-2xl font-medium mb-8">Why give to WNB.rb?</h2>
+                    <p className="mb-5">
+                        Donating to WNB.rb enables us to invest more resources in supporting women
+                        and non-binary people in the Ruby community.
+                    </p>
+                    <p>
+                        Here are some of the initiatives we plan to launch in 2022 with your help:
+                    </p>
+                    <ul className="list-disc ml-10">
+                        <li>Community fund for educational materials</li>
+                        <li>Full scholarships to RubyConf and RailsConf</li>
+                        <li>Open source development stipends</li>
+                    </ul>
+                </div>
+            </div>
+            <OtherAmountBanner />
+        </SharedLayout>
+    );
+};
+
+const DonationAmounts = ({ selectedPrice, setSelectedPrice }) => {
+    const prices = useMemo(() => {
+        return process.env.NODE_ENV === 'development'
+            ? testDonationAmounts
+            : productionDonationAmounts;
+    }, []);
+
+    const selectedPriceObject = useMemo(() => {
+        return prices.filter((price) => {
+            return price.value === selectedPrice;
+        })[0];
+    }, [prices, selectedPrice]);
+
+    return (
+        <div className="flex justify-center">
+            <div className="donation-amounts">
+                {prices.map((price) => {
+                    return (
+                        <button
+                            key={price.value}
+                            className={`donation-amount ${
+                                selectedPrice === price.value ? 'selected' : null
+                            }`}
+                            onClick={() => setSelectedPrice(price.value)}
+                        >
+                            ${price.value.toLocaleString()}
```

Comment:
@jonsamp taught me about `toLocaleString`, which formats the price number correctly (e.g. 1000 becomes 1,000).